src/coreclr/src/pal/src/map/map.cpp

```diff
@@ -2441,19 +2457,25 @@ void * MAPMapPEFile(HANDLE hFile, off_t offset)
     //we have now reserved memory (potentially we got rebased).  Walk the PE sections and map each part
     //separately.
 
+    _ASSERTE(OffsetWithinPage((off_t)loadedBase) == 0);
 
     //first, map the PE header to the first page in the image.  Get pointers to the section headers
+    loadedHeader = (IMAGE_DOS_HEADER*)(static_cast<char*>(loadedBase) + OffsetWithinPage(offset));
+
+    LPVOID loadedHeaderBase;
+    loadedHeaderBase = NULL;
```

Comment:
Why two lines to initialize?

Reply:
It was in the original change. Maybe something more was there and later removed.
I will change this.

Reply:
Ahaa:  `2465:12: note: jump bypasses variable initialization`

Reply:
I guess this will have to be on two lines